python/vtool/script_manager/script_view.py

```diff
@@ -49,16 +49,15 @@ def _load_code(self, filepath=None):
             self.tree.manager_widget.run_script.hide()
             return
 
-        if not filepath:
-            self.tab_widget.setTabEnabled(1, True)
-            filepath = self.tree.get_current_item_directory()
-            self.tree.manager_widget.run_script.show()
-
         if filepath:
             if not util_file.is_file(filepath):
                 self.tab_widget.setTabEnabled(1, False)
                 self.tree.manager_widget.run_script.hide()
                 return
+        else:

```

Comment:
This may have introduced a bug. The previous code was looking for an absence of filepath, assigning a value, and then it was being checked for again.